package.json

```diff
@@ -51,14 +55,39 @@
     "typescript": "^4.8.3"
   },
   "scripts": {
+    "about:start": "echo 'Start the validator, as a TypeScript command line application'",
+    "start": "npx ts-node src/main.ts",
+    "about:eslint": "echo 'Perform linting of the source code. This may modify the source files.'",
     "eslint": "eslint \"./**/*.ts\" --cache",
+    "about:build": "echo 'Run the TypeScript compiler to generate the build output.'",
     "build": "npx tsc",
-    "start": "npx ts-node src/main.ts",
+    "about:test": "echo 'Execute the unit tests'",
     "test": "ts-node node_modules/jasmine/bin/jasmine --config=specs/jasmine.json",
+    "about:coverage": "echo 'Compute the test coverage'",
     "coverage": "nyc npm run test",
+    "about:prettier-check": "echo 'Check the formatting of the source code (without fixing formatting errors).'",
     "prettier-check": "prettier --check \"**/*\"",
+    "about:prettier": "echo 'Ensure the right formatting of the source code, automatically fixing formatting errors.'",
     "prettier": "prettier --write \"**/*\"",
+    "about:generate-third-party": "echo 'Update the ThirdParty.json file with the most recent dependency versions'",
     "generate-third-party": "node generateThirdParty.js",
-    "docs": "mkdirp etc && npx tsc && api-extractor run --config api-extractor.jsonc --local --verbose && api-documenter markdown -i build/api-extractor -o build/docs"
+    "about:docs-prepare-directory": "echo 'Prepare the output directory for the API definition'",
+    "docs-prepare-directory": "mkdirp etc",
+    "about:docs-extract-api": "echo 'Extract the API definition file from the current build output.'",
+    "docs-extract-api": "api-extractor run --config api-extractor.jsonc --local --verbose",
+    "about:docs-generate-markdown": "echo 'Generate the API documentation based on the API definition file'",
+    "docs-generate-markdown": "api-documenter markdown -i build/api-extractor -o build/docs",
+    "about:docs-generate": "echo 'Generate the API documentation based on the current build output'",
+    "docs-generate": "npm run docs-prepare-directory && npm run docs-extract-api && npm run docs-generate-markdown",
+    "about:docs": "echo 'Perform a build and generate the API documentation based on the build output'",
+    "docs": "npm run build && npm run docs-generate",
+    "about:package-clean": "echo 'Clean the build output folder'",
+    "package-clean": "node -e \"require('fs-extra').emptyDir('build');\"",
```

Comment:
`fs-extra` should be explicitly included in `devDependencies`. Also `emptyDirSync` is better to use here.

Is there a simpler way to do this? Maybe https://www.npmjs.com/package/del-cli?